src/Tizen.NUI.WindowSystem/src/public/SoftkeyClient.cs

```diff
@@ -0,0 +1,336 @@
+        }
+
+        /// <summary>
+        /// Requests to hide the softkey service window.
+        /// </summary>
+        /// <exception cref="ArgumentException" > Thrown when failed of invalid argument.</exception>
+        /// <exception cref="NotSupportedException">Thrown when the feature is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when failed because of an invalid operation or no service.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public void Hide()
+        {
+            int res;
+
+            res = Interop.SoftkeyClient.Hide(_softkeyClient);
+            _tzsh.ErrorCodeThrow(res);
+        }
+
+        /// <summary>
+        /// Gets the visible state of a softkey service window.
+        /// </summary>
+        /// <returns>The visible state of the softkey service window.</returns>
+        /// <exception cref="ArgumentException" > Thrown when failed of invalid argument.</exception>
+        /// <exception cref="NotSupportedException">Thrown when the feature is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when failed because of an invalid operation or no service.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public VisibleState Visible
+        {
+            get
+            {
+                return GetVisible();
+            }
+        }
+
+        private VisibleState GetVisible()
+        {
+            int res;
+            int vis;
+
+            res = Interop.SoftkeyClient.GetVisibleState(_softkeyClient, out vis);
+
+            _tzsh.ErrorCodeThrow(res);
+
+            return (VisibleState)vis;
+        }
+
+        /// <summary>
+        /// Gets the expand state of a softkey service window.
+        /// </summary>
+        /// <returns>The expand state of the softkey service window.</returns>
+        /// <exception cref="ArgumentException" > Thrown when failed of invalid argument.</exception>
+        /// <exception cref="NotSupportedException">Thrown when the feature is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when failed because of an invalid operation or no service.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public ExpandState Expand
+        {
+            get
+            {
+                return GetExpand();
+            }
+            set
+            {
+                SetExpand(value);
+            }
+        }
+
+        private ExpandState GetExpand()
+        {
+            int res;
+            int expand;
+
+            res = Interop.SoftkeyClient.GetExpandState(_softkeyClient, out expand);
+
+            _tzsh.ErrorCodeThrow(res);
+
+            return (ExpandState)expand;
+        }
+
+        private void SetExpand(ExpandState expand)
+        {
+            int res;
+
+            res = Interop.SoftkeyClient.SetExpandState(_softkeyClient, (int)expand);
+
+            _tzsh.ErrorCodeThrow(res);
```

Comment:
Unnecessary declaration line
```suggestion
            int res = Interop.SoftkeyClient.SetExpandState(_softkeyClient, (int)expand);
            _tzsh.ErrorCodeThrow(res);
```

Reply:
I changed it.